engine_comparison/README.md

```diff
@@ -4,15 +4,36 @@ This is a set of scripts to run A/B testing among different fuzzing engines.
 
 ## gcloud
 
-The `gcloud` CLI is a part of the [Google Cloud SDK](https://cloud.google.com/sdk/gcloud/), which can be installed [here](https://cloud.google.com/sdk/downloads). 
+The `gcloud` CLI is a part of the [Google Cloud SDK](https://cloud.google.com/sdk/gcloud/), which can be installed [here](https://cloud.google.com/sdk/downloads).
 
 Currently, these scripts only run on Google Cloud, but support for alternatives will be incorporated.
 
 ## Usage
 
-From one's local computer, call ` ${FTS}/engine_comparison/begin_experiment.sh`
+From one's local computer, call ` ${FTS}/engine_comparison/begin_experiment.sh
+<list of benchmarks> <fuzzing engine configuration> <more configurations>...`
```

Comment:
<more configurations> is a bit confusing.  Would be more clear if you had something like `${FTS}/engine_comparison/begin_experiment.sh <benchmarks> <fuzzing-config1> <fuzzing-config2> ...`